src/features/settings/screens/UsersScreen/UsersScreen.module.scss

```diff
@@ -1,7 +1,11 @@
 @import "styles/variables";
 
-.buttonContainer {
-  display: flex;
-  justify-content: flex-end;
-  margin: $spacing-4;
+.table {
```

Comment:
this should be moved inside the `UsersTable` so it is closer to where it is actually being used